jbcam/dotfiles
# merge upstream

- add git plugin to restore gitfast aliases
- Fix typo on Sublime Text setting
- Remove gitfast plugin
- NVM + Anaconda in PATH
- Latest package install
- Anaconda prompt
- Prepare setup for Data Bootcamp (pyenv / virtualenv). Should not conflict with Web Bootcamp
- Fix rbenv + pyenv on Linux
- add `BracketHighlighter` as default setup pack
- Update Package Control.sublime-settings
- Change default aliases
- Fix color highlight package
- Update zshrc comment
- ignore Python
- update nvm load
- silence nvm to remove unecesary stdout
- Add default pull reconciliation strategy for git
- remove virtualenv auto-loading
- Disable PYENV_VIRTUALENV_DISABLE_PROMPT
- Add Python activated environment to the right prompt
- add anaconda package to sublime text
- add flake8 linter and AutoPEP8 formatter
- git clone zsh autosuggestions
- sort sublime package list
- update nvm path
- reverting to usual NVM dir
- Fix missing comma in stt packlist
- load only if nvm is installed
- add vscode settings
- updating install.sh
- remove package control
- remove sublime linter sublime settings
- update install.sh
- fix bug and restore backup command of vscode settings
- override existing symlink
- exclude vscode settings from home symlinks
- update vscode_settings.json
- restore zshrc options
- update vscode settings
- add disable git to vscode settings
- replace syntax highlighting extension protocol
- remove .git extension
- add ssh passphrase config file
- adding a new dotfile for ssh passphrase management on macos
- refactor install.sh
- import default vscode settings.json
- update config
- remove unused files and ssh-add for macos
- update code path for wsl
- update wsl code path
- add code as editor in gitconfig
- remove git config editor command and add the corresponding line in gitconfig file
- protect load_nvmrc call when nvm is not installed yet
- add env var to disable warning
- update Darwin
- protect load nvm
- remove load-nvmrc precondition
- add if nvm in load-nvmrc script
- fix typo
- add &
- fix ssh key name
- update condition
- remove temp echo
- force to redo the symlink if already exists
- refactor symlink function
- fix ssh key file name
- dotfiles are for web and data students
- merge settings from web dev and data science
- remove sublime setting
- fixes #89
- add a profile to the setup in order to handle the PATH for pyenv
- protect pyenv command for web setup
- Remove Ruby extension settings messing with the editor
- rename profile into zprofile so that it is sourced by zsh
- remove export PATH from zshrc
- disable virtual env prompt
- add quoting around git email and full_name, to make the script less likely to break in the case of bad output
- add emmet option then format and sort json
- add bracket pairs
- add bracket pair colorization setting
- update conf to have vscode env match pyenv according to https://github.com/lewagon/data-setup/issues/156
- restore python path
- add vs code keybindings
- add EDITOR env var in zshrc
- Update settings.json
- sort json
- add python specific vscode settings
- Update settings.json
- converge and sort
- use ipdb as the default Python debugger
- add wait flag
- add default branch master to gitconfig
- update vscode setup
- Disable lt alias in favor of https://github.com/localtunnel/localtunnel (ngrok alternative)
- Update settings.json
- add default size for vscode to 'maximized'
- push default terminal position to right
- My identity for @lewagon in the gitconfig
- updated aliases
- removed sublime config
- added VS code as main editor
- updates
- updated gut aliases
- update
- log one line
- added git auto fetch
- gp
- fix
- added push --force-with-lease alias
- added up script
- add vscode settings.json
